gen database revised for CEJA and state cap

ssattler opened the pull request:
revised EIA-NEMS database to reflect CEJA, and revised state cap file to implement state-level carbon cap.

From James Gignac: https://ucsusa-my.sharepoint.com/:x:/r/personal/jgignac_ucs_org/_layouts/15/Doc.aspx?sourcedoc=%7B94B54D83-D789-4D97-829B-B5F5547BF70A%7D&file=CEJA%20plant%20retirements%20for%20ReEDS.xlsx&action=default&mobileredirect=true

The following plants are already retired:
1515 South Caron Road – Retired March 2022
Freedom Power Project – Retired May 2024
Grand Tower Energy C – Retired Sept. 2023
Southeast Chicago En – Retired March 2024

Morris Cogeneration – Set retirement for 2045

ssattler committed:
gen database revised for CEJA and state cap

revised EIA-NEMS database to reflect CEJA, and revised state cap file to implement state-level carbon cap.

ssattler committed:
revised state_cap file for total emissions

Revised the declining state cap for MI and WI to reflect total emissions (not just emissions from in state generation)
2023 emissions for our states of interest are:

Total emissions in Michigan: 50449297.96635092
Total emissions in Wisconsin: 49933301.84715779
Total emissions in Louisiana: 46108734.053563096

samgdotson commented on inputs/emission_constraints/state_cap.csv:
```diff
@@ -1,206 +1,206 @@
+MI,2020,100000000
+MI,2021,100000000
+MI,2022,100000000
+MI,2023,100000000
+MI,2024,100000000
+MI,2025,100000000
+MI,2026,47554062.92
```
I think we should start the emissions cap in 2023 based on the values I sent. For MI, the 2023 value should be 50449297.97

samgdotson commented on inputs/emission_constraints/state_cap.csv:
```diff
@@ -1,206 +1,206 @@
+WI,2020,100000000
+WI,2021,100000000
+WI,2022,100000000
+WI,2023,100000000
+WI,2024,100000000
+WI,2025,100000000
+WI,2026,28211188.66
+WI,2027,26196103.76
+WI,2028,24181018.85
+WI,2029,22165933.95
```
I'm not sure this trajectory matches a linearly reducing cap starting at 49933301.85 in 2023.

samgdotson commented on inputs/emission_constraints/state_cap.csv:
```diff
@@ -1,206 +1,206 @@
+LA,2020,100000000
+LA,2021,100000000
+LA,2022,100000000
+LA,2023,100000000
+LA,2024,100000000
+LA,2025,100000000
+LA,2026,32875013.39
+LA,2027,30776608.28
```
The cap for LA should start at 46108734 in 2023

samgdotson reviewed (changes requested):
I think the state cap needs to be updated but the NEMS database looks good

ssattler committed:
corrected the state cap trajectories

I had an error in my spreadsheet, here are the correct total state emissions cap trajectories.

ssattler commented:
I had an error in the spreadsheet where I calculated the emissions trajectories - these state emissions caps should be correct now.

samgdotson reviewed (approved):
Looks good to me!
